Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/InBetween.cs

```diff
@@ -27,35 +27,49 @@ public float PartwayOffset
         [SerializeField]
         [Tooltip("Tracked object to calculate position and orientation for the second object. If you want to manually override and use a scene object, use the TransformTarget field.")]
         [HideInInspector]
-        private TrackedObjectType trackedObjectForSecondTransform = TrackedObjectType.Head;
+        private TrackedObjectType secondTrackedObjectType = TrackedObjectType.Head;
 
         /// <summary>
         /// Tracked object to calculate position and orientation for the second object. If you want to manually override and use a scene object, use the TransformTarget field.
         /// </summary>
-        public TrackedObjectType TrackedObjectForSecondTransform
+        public TrackedObjectType SecondTrackedObjectType
```

Comment:
>SecondTrackedObjectType [](start = 33, length = 23)

Please add a migration note in the GA doc for this change.

I'm kinda torn between keeping things as deprecated and having them reference this - it might be good to be consistent with that (i.e. in all of these public field renames, keep the old around, mark as deprecated, put it as a task on me to do the post-deprecation deletion process). What do you think?

Reply:
I think that is reasonable. It's more work on our side as developers but it should make it easier for consumers. They can replace all references in their framework to the new name probably more easily since VS can still resolve the reference